base/BUILD.gn

```diff
@@ -1561,7 +1561,7 @@ component("base") {
     public_configs = [ ":base_public_defines" ]
     deps += [
       "//starboard/common",
-      "//starboard",
```

Comment:
I don't think we should have that level of implicitness, target renaming under the hood would just cause more confusion down the line. Is the worry that people will depend on the wrong target? If so, we could potentially solve that with visibility settings

Reply:
i had done that before using "output_name" but andrew's suggestion was to not change that and instead change the name of the target itself to starboard rather than starboard_platform_group